docs/configuration.asciidoc

```diff
@@ -36,20 +36,22 @@ With this you can use any configuration source that you configured on the `IConf
 
 NOTE: By simply calling `app.UseElasticApm()` without the overload, the agent will read configurations only from environment variables.
 
+NOTE: For better user experience, `LogLevel` for internal apm logger is used from standard `Logging` section inside `appsettings.json` with `Apm` category name.
+
 This is a typical `appsettings.json` file that contains some sample configuration. The part below `ElasticApm` is fetched by the agent if the corresponding `IConfiguration` is passed to the agent.
 
 [source,js]
 ----
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Warning"
+      "Default": "Warning",
+      "Apm": "Debug"
     }
   },
   "AllowedHosts": "*",
   "ElasticApm":
     {
-      "LogLevel":  "Debug",
```

Comment:
Additionally I'd add another block below this one to explain how to use `ElasticApm:LogLevel` when ASP.NET Core is not involved. 

Something like this:

In case you don't activate the agent with the `UseElasticApm()` method (typically non-ASP.NET Core scenario) then you can set the loglevel of the agent with the `ElasticApm:LogLevel` setting, as shown in the following `appsettings.json` file:

[and then use the original file]

(_feel free to adapt or improve the text if you feel something is not ok_)

Reply:
added